.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## Add new chain
+
+Template to provide information about the new chain. Only add extra information in the bottom section. Ensure that the contracts are deployed on the chain, if not deploy with [safe-contracts](https://github.com/safe-global/safe-contracts). This entire paragraph can be deleted.
+
+Please fill the following form:
+
+Provide the Chain ID (Only 1 chain id per PR).
+- Chain_ID: 
+
+Provide RPC URL for the chain (should be able to query atleast 10 requests per minute for automatic PR check).
+- RPC_URL: 
+
+What version (1.3.0/1.4.1) of contracts are deployed? NOTE: One issue per version and one pull request per version. Do not add `v` infront of the version.
+- Contract_Version: 
+
+Relevant information:
+Add any other relevant information like blockexplorer, any annotation...
```

Comment:
I think templates have a "form" system - see https://github.com/safe-global/safe-singleton-factory/blob/main/.github/ISSUE_TEMPLATE/new_chain.yml

This _may_ apply to pull request templates, but I'm not 100% sure.

Reply:
I did try with form structure, but it only works for Issue, and didn't for PR Templates.

Reply:
Only `.md` and `.txt` are supported: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates#pull-request-templates